src/components/HourlyMeteo.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+
+import styles from './HourForecast.module.css';
+
+const api_weather = 'a007f6ea1885d8331305baf19e99c488';
+
+function HourlyMeteo({ coordinates, destination }) {
+  const [forecast, setForecast] = useState([]);
+  const lat = coordinates.lat;
+  const long = coordinates.lon;
+
+  useEffect(async () => {
+    const result = await axios.get(
+      `https://api.openweathermap.org/data/2.5/onecall?lat=${lat}&lon=${long}&appid=${api_weather}&units=metric&lang=fr&exclude=minutely`,
+    );
+    let res = result.data;
+    setForecast(res.hourly);
+  }, [coordinates, destination]);
+
+  let forecastHours = forecast.slice(0, 8);
```

Comment:
Transform this let in a state and apply the slice inside a useEffect 
```suggestion
const [forcastHours, setForecastHours] = useState([]);

useEffect(() => {
  setForcastHours(forcast.slice(0, 8);
}, [forcast])
```